src/CommunityPatch/ApplicationVersionComparer.cs

```diff
@@ -25,8 +25,10 @@ public class ApplicationVersionComparer : IComparer<ApplicationVersion>, IEquali
     public bool Equals(ApplicationVersion x, ApplicationVersion y)
       => Compare(x, y) == 0;
 
+    public bool GreaterThan(ApplicationVersion x, ApplicationVersion y) => Compare(x, y) > 0;
+
     public int GetHashCode(ApplicationVersion obj)
-      => ((int) _typeComparer.GetHashCode(obj.ApplicationVersionType) << 28) | (obj.Major << 23) | (obj.Minor << 12) | obj.Revision;
+    => ((int) _typeComparer.GetHashCode(obj.ApplicationVersionType) << 28) | (obj.Major << 23) | (obj.Minor << 12) | obj.Revision;
 
   }
```

Comment:
Can you revert these minor formatting changes, they might be accidentally committed?
They're out of scope for the PR.

Reply:
Fixed

Reply:
This shouldn't show up in the PR as modified...